kit-slides.typ

```diff
@@ -22,6 +22,8 @@
 #let yellow = rgb(252, 229, 0)
 #let orange = rgb(223, 155, 27)
 #let red = rgb(162, 34, 35)
+#let lightgreen = rgb(140, 182, 60)
```

Comment:
This is already there as `lime`.
It corresponds to the color "Maigrün" in the KIT corporate design.